win_build_vs.bat

```diff
@@ -108,9 +108,9 @@ mkdir %builddir%
 pushd %builddir%\
 
 if [%generator%] EQU ["Ninja"] (
-    set "buildcmd=ninja && copy bin\minifi.exe main\"
+    set "buildcmd=ninja && copy bin\minifi.exe minifi_main\"
 ) else (
-    set "buildcmd=msbuild /m nifi-minifi-cpp.sln /property:Configuration=%cmake_build_type% /property:Platform=%build_platform% && copy bin\%cmake_build_type%\minifi.exe main\"
+    set "buildcmd=msbuild /m nifi-minifi-cpp.sln /property:Configuration=%cmake_build_type% /property:Platform=%build_platform% && copy bin\%cmake_build_type%\minifi.exe minifi_main\"
```

Comment:
what is the purpose of the copy command here? is it required to run the tests?

Reply:
It is required by `cpack`, when creating the `msi` installer.

Reply:
I see, thanks